Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml

```diff
@@ -43,7 +43,7 @@
         - to: reinforcedGirder
           completed:
             - !type:SnapToGrid
-              southRotation: true
+              southRotation: false
```

Comment:
Why was this changed? This will affect ordinary walls, too. Why do shuttle walls need special-casing?

Reply:
It's the diagonal walls that need this. If they snap false, it un-rotates the construction ghost. 
![image](https://github.com/space-wizards/space-station-14/assets/107660393/c2516eeb-d262-41f7-9684-7e62897ace99)
![image](https://github.com/space-wizards/space-station-14/assets/107660393/1b3610ca-c2f3-49c8-8989-f719f1f75b28)

W/ southRotation set to false:
![image](https://github.com/space-wizards/space-station-14/assets/107660393/f2b35319-e661-4245-b8bd-e57059a000a7)